package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@internetarchive/your-webcomponent",
-  "version": "0.0.0",
+  "name": "@internetarchive/throttler-debouncer",
+  "version": "0.0.1",
   "description": "The Internet Archive Typescript WebComponent template. It is based on the @open-wc template.",
```

Comment:
Make sure to update the description here.

Reply:
updated